frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -28,26 +28,31 @@ end
 -- Load the user dictionary suitable for the actual language
 -- if reload==true, force a reload
 -- Unload is done automatically when a new dictionary is loaded.
-function ReaderUserHyph:loadDictionary(name, reload)
+function ReaderUserHyph:loadDictionary(name, reload, no_scrubbing)
     if G_reader_settings:isTrue("hyph_user_dict") and lfs.attributes(name, "mode") == "file" then
-        logger.dbg("set user hyphenation dict", name, reload)
+        logger.dbg("set user hyphenation dict", name, reload, no_scrubbing)
         local ret = cre.setUserHyphenationDict(name, reload)
         -- this should only happen, if a user edits a dictionary by hand or the user messed
         -- with the dictionary file by hand. -> Warning and disable.
         if ret == self.USER_DICT_ERROR_NOT_SORTED then
-            UIManager:show(InfoMessage:new{
-                text = T(_("The user dictionary\n%1\nis not alphabetically sorted.\n\nIt has been disabled."), name),
-            })
-            logger.warn("UserHyph: Dictionary " .. name .. " is not sorted alphabetically.")
-            G_reader_settings:makeFalse("hyph_user_dict")
+            if no_scrubbing then
+                UIManager:show(InfoMessage:new{
+                    text = T(_("The user dictionary\n%1\nis not alphabetically sorted.\n\nIt will be disabled now."), name),
```

Comment:
Is it possible that after "scrubbing", you'd still get the user dict not sorted ?

Reply:
I hope that we never see that message. If we see that message, we have a bug: Either in `scrubDictionary()` or in crengine's `UserHyphDict::addEntry`.